### PR TITLE
Run black on each commit with travis

### PR DESCRIPTION
**Pull request by kinnou02**
[skip ci]

### CODE REVIEW

**Review by nonifier — APPROVED** [review]
:black_heart:

File: .travis.yml [hunk 1]
```diff
@@ -10,13 +10,15 @@ git:
   submodules: false
 
 before_install:
-  - sudo apt update && sudo apt install -y python-pip python-dev protobuf-compiler
+  - sudo apt update && sudo apt install -y python-pip python-dev protobuf-compiler python3-pip
   - sed -i 's,git\@github.com:\([^/]*\)/\(.*\).git,https://github.com/\1/\2,' .gitmodules
   - git submodule update --init --recursive
   - sudo pip install -r source/tyr/requirements_dev.txt
+  - pip3 install black
```
Comment (nonifier) [inline .travis.yml]: from what I remember, pip3 uses python 3.5 when Black requires 3.6 :/ 
You need to do something like: `python3.6 -m pip install black`

File: pyproject.toml [hunk 2]
```diff
@@ -11,5 +11,6 @@ exclude = '''
   | debug
   | debian
   | source/third_party
-)/
+  | source/navitiacommon/navitiacommon/.*_pb2\.py
```
Comment (pbougue) [inline pyproject.toml]: You also have chaos proto

**Review by pbougue — COMMENTED** [review]
Weird black behavior, not sure I get the same for:
* `pre-commit run black --all` vs `black .`
* on my laptop vs yours (cf my https://github.com/CanalTP/navitia/pull/2562)

**Comment by kinnou02** [comment]
Jenkins, test this please.

File: pyproject.toml (hunk 2 above)
Comment (kinnou02) [reply pyproject.toml]: I did fix the version used for black in this PR, some of us didn't have the same version.

File: .travis.yml [hunk 3]
```diff
@@ -10,13 +13,15 @@ git:
   submodules: false
 
 before_install:
-  - sudo apt update && sudo apt install -y python-pip python-dev protobuf-compiler
+  - sudo apt update && sudo apt install -y protobuf-compiler
   - sed -i 's,git\@github.com:\([^/]*\)/\(.*\).git,https://github.com/\1/\2,' .gitmodules
   - git submodule update --init --recursive
-  - sudo pip install -r source/tyr/requirements_dev.txt
+  - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then pip install -r source/tyr/requirements_dev.txt; fi
+  - if [[ $TRAVIS_PYTHON_VERSION == '3.6' ]]; then python3.6 -m pip install black==18.9b0; fi
```
Comment (nonifier) [inline .travis.yml]: Out of curiosity, how do you tell Travis to use either 2.7 or 3.6 ?

File: source/jormungandr/jormungandr/interfaces/v1/serializer/api.py [hunk 4]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (pbougue) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/api.py:1]: Here in py PR I prefered adding `coding=utf-8` and keep that line... Anyway I'll rebase mine and it should work fine.

**Review by pbougue — APPROVED** [review]
Anyway it's fine for now, and we'll adjust if we have so problems between `pre-commit` and `black .`

**Pull request merged**